### PR TITLE
Create Dockerfile

### PR DESCRIPTION
**Pull request by synost**
Added a Dockerfile that support both amd64 and arm64.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+WORKDIR /build
+
+COPY Cargo.toml Cargo.lock ./
+COPY src ./src
+
+RUN cargo fetch
+RUN cargo build --release
+
+FROM debian:bookworm-slim as dependencies
```
Comment (alexheretic) [inline Dockerfile:11]: How come this is `as dependencies` then later `as runtime` wouldn't it be the same to just have it `as runtime` here?

**Comment by alexheretic** [comment]
Thanks this seems nice to have. Perhaps we could also add CI to build the image on new tags, something a bit like https://github.com/alexheretic/aurto/blob/main/.github/workflows/build.yml

File: Dockerfile (hunk 1 above)
Comment (synost) [reply Dockerfile]: You right, it's basically the same. I've removed the dependencies stage

**Comment by alexheretic** [comment]
Thanks!Llets add CI to build this on tag sometime later.

**Pull request merged**